bot/utils/blurple_formatter.py

```diff
@@ -598,6 +711,7 @@ def unparse(node: ast.AST, nl_able: bool = False) -> str:
         return (
             f"({space()}"
             + f"{space()},{space()}".join(unparse(elt, True) for elt in elts)
+            + f"{space()},{space()}" * (len(elts) == 1)
             + f"{space()})"
```

Comment:
With the second line in mind, won't this give us duplicates trailing space?

Reply:
Gonna fix.